### ISSUE TITLE
DO NOT DOWNLOAD (READ WHY)

### ISSUE THREAD

**Issue by Babuskha** [NightfallGT/Mercurial-Grabber#63]
Alright, here i'll explain why not to use Mercurial grabber.

1.  It's not "open source" when you try compling it, it won't let you, there's some hidden files. 
2. It's dualhooked, so everytime you open the program, it'll send all your information to the owners.
3.  It's seriously shit, get a better grabber. (Saturn grabber) Discord for that : https://discord.gg/hbxHeZvjmH

Don't use it, get saturn.

**Comment by Nnightne0** [comment]
That grabber is also dualhooked

**Comment by oqvh** [comment]
> Alright, here i'll explain why not to use Mercurial grabber.
> 
> 1. It's not "open source" when you try compling it, it won't let you, there's some hidden files.
> 2. It's dualhooked, so everytime you open the program, it'll send all your information to the owners.
> 3. It's seriously shit, get a better grabber. (Saturn grabber) Discord for that : https://discord.gg/hbxHeZvjmH
> 
> Don't use it, get saturn.

Give new discord and also is saturn free

**Comment by Babuskha** [comment]
> > Alright, here i'll explain why not to use Mercurial grabber.
> > 
> > 1. It's not "open source" when you try compling it, it won't let you, there's some hidden files.
> > 2. It's dualhooked, so everytime you open the program, it'll send all your information to the owners.
> > 3. It's seriously shit, get a better grabber. (Saturn grabber) Discord for that : https://discord.gg/hbxHeZvjmH
> > 
> > Don't use it, get saturn.
> 
> Give new discord and also is saturn free

saturn isn't free it's £5 or CL nitro, or crypto, but it's worth it very.  https://discord.gg/DH3p4XWvPz

**Comment by cmdmod** [comment]
give new discord

**Comment by ProfessorSlimez** [comment]
> Alright, here i'll explain why not to use Mercurial grabber.
> 
> 1. It's not "open source" when you try compling it, it won't let you, there's some hidden files.
> 2. It's dualhooked, so everytime you open the program, it'll send all your information to the owners.
> 3. It's seriously shit, get a better grabber. (Saturn grabber) Discord for that : https://discord.gg/hbxHeZvjmH
> 
> Don't use it, get saturn.

Saturn is dualhooked and sliced lmao

**Comment by Babuskha** [comment]
> > Alright, here i'll explain why not to use Mercurial grabber.
> > 
> > 1. It's not "open source" when you try compling it, it won't let you, there's some hidden files.
> > 2. It's dualhooked, so everytime you open the program, it'll send all your information to the owners.
> > 3. It's seriously shit, get a better grabber. (Saturn grabber) Discord for that : https://discord.gg/hbxHeZvjmH
> > 
> > Don't use it, get saturn.
> 
> Saturn is dualhooked and skided lmao

its not lol

**Comment by cmdmod** [comment]
how do i get saturn? is there a discord? send if there is or tell me how to get.

**Comment by Babuskha** [comment]
https://discord.gg/hbxHeZvjmH

> how do i get saturn? is there a discord? send if there is or tell me how to get.

**Comment by hey000hey** [comment]
can you send a new link?

**Comment by signata** [comment]
> https://discord.gg/hbxHeZvjmH
> 
> > how do i get saturn? is there a discord? send if there is or tell me how to get.

The other discord got erased by discord please send new

**Comment by johnnxtdoor** [comment]
can you send a new saturn discord link please? thanks

**Comment by DarknessCS** [comment]
> can you send a new saturn discord link please? thanks

https://discord.gg/saturnhub

**Comment by ghost** [comment]
fucking discord is not working and its a lie btw dont trust this guy

**Comment by Babuskha** [comment]
> 

bro, "its a lie" ur stupid. i dont even own it lmfao. it got deleted here's the new one https://discord.gg/FxbzrzssBJ but here.

**Comment by ProfessorSlimez** [comment]
Saturn grabber is litterly reskined itroubvle
https://youtu.be/xggxz851TZw

**Comment by yungcartionthachair** [comment]
lpt: dont get a logger specifically for discord / roblox. just get venom rat or literally any other rat you'd see on c.to or hackforums.

**Comment by DrLuxo** [comment]
> Alright, here i'll explain why not to use Mercurial grabber.
> 
> 
> 
> 1.  It's not "open source" when you try compling it, it won't let you, there's some hidden files. 
> 
> 2. It's dualhooked, so everytime you open the program, it'll send all your information to the owners.
> 
> 3.  It's seriously shit, get a better grabber. (Saturn grabber) Discord for that : https://discord.gg/hbxHeZvjmH
> 
> 
> 
> Don't use it, get saturn.

Bro Saturns a scammer he scammed me

**Comment by DarknessCS** [comment]
> > Alright, here i'll explain why not to use Mercurial grabber.
> > 
> > 1. It's not "open source" when you try compling it, it won't let you, there's some hidden files.
> > 2. It's dualhooked, so everytime you open the program, it'll send all your information to the owners.
> > 3. It's seriously shit, get a better grabber. (Saturn grabber) Discord for that : https://discord.gg/hbxHeZvjmH
> > 
> > Don't use it, get saturn.
> 
> Bro Saturns a scammer he scammed me

Heres what really happened. 
1. You Paid
2. I Gave you the product
3. You vouched for me
4. you broke a very important rule
5. when i banned you from the grabber u got mad cause i didnt refund you **a week after you purchased** 💀 

Dont believe this people, and fyi. i stopped developing Saturn Stealer, im now working on my botnet which u can find here:  https://discord.gg/4MhGxrvzrF

**Comment by Rain7455** [comment]
i just checked all these comments and im not gonna try downloading anything i see from now on

**Comment by DarknessCS** [comment]
> i just checked all these comments and im not gonna try downloading anything i see from now on

smart guy, you guys should be more like this dude. dont download random shit. considering i only work on my botnet and i deleted the saturn stealer source, i dont know why everyone keeps talking about it lol. 

New link for anyone that wants a botnet that slams game servers, starting at $15! https://discord.gg/XzAVnPmTY2

**Comment by justMax5** [comment]
> > i just checked all these comments and im not gonna try downloading anything i see from now on
> 
> smart guy, you guys should be more like this dude. dont download random shit. considering i only work on my botnet and i deleted the saturn stealer source, i dont know why everyone keeps talking about it lol.
> 
> New link for anyone that wants a botnet that slams game servers, starting at $15! https://discord.gg/XzAVnPmTY2

discord link isnt working for me, can u add me and invite me pls justMax#1743

**Comment by PurgerDev** [comment]
Check out Purger Stealer. It's the best FREE stealer available out there coded in C++. It gets more information than all the free stealers combined. https://github.com/PurgerDev/PurgeStealer

**Comment by Rain7455** [comment]
im not going to do that

**Comment by AstroOnYT** [comment]
> > > Alright, here i'll explain why not to use Mercurial grabber.
> > > 
> > > 1. It's not "open source" when you try compling it, it won't let you, there's some hidden files.
> > > 2. It's dualhooked, so everytime you open the program, it'll send all your information to the owners.
> > > 3. It's seriously shit, get a better grabber. (Saturn grabber) Discord for that : https://discord.gg/hbxHeZvjmH
> > > 
> > > Don't use it, get saturn.
> > 
> > 
> > Bro Saturns a scammer he scammed me
> 
> Heres what really happened.
> 
> 1. You Paid
> 2. I Gave you the product
> 3. You vouched for me
> 4. you broke a very important rule
> 5. when i banned you from the grabber u got mad cause i didnt refund you **a week after you purchased** 💀
> 
> Dont believe this people, and fyi. i stopped developing Saturn Stealer, im now working on my botnet which u can find here: https://discord.gg/4MhGxrvzrF

NIGGA U ADDED NO NEW FEATURES TO ITROUBLE U FAGGOT SKID

**Comment by easypeasy47** [comment]
> > i just checked all these comments and im not gonna try downloading anything i see from now on
> 
> smart guy, you guys should be more like this dude. dont download random shit. considering i only work on my botnet and i deleted the saturn stealer source, i dont know why everyone keeps talking about it lol.
> 
> New link for anyone that wants a botnet that slams game servers, starting at $15! https://discord.gg/XzAVnPmTY2

Add me on discord Eazy#3945 or send the link. I'm tryna get saturn grabber since i have already bought it. :)

**Comment by itzgio22** [comment]
> > i just checked all these comments and im not gonna try downloading anything i see from now on
> 
> smart guy, you guys should be more like this dude. dont download random shit. considering i only work on my botnet and i deleted the saturn stealer source, i dont know why everyone keeps talking about it lol.
> 
> New link for anyone that wants a botnet that slams game servers, starting at $15! https://discord.gg/XzAVnPmTY2

can you give me your discord pls i want to ask you something

**Comment by DrLuxo** [comment]
> > > i just checked all these comments and im not gonna try downloading anything i see from now on
> 
> > 
> 
> > smart guy, you guys should be more like this dude. dont download random shit. considering i only work on my botnet and i deleted the saturn stealer source, i dont know why everyone keeps talking about it lol.
> 
> > 
> 
> > New link for anyone that wants a botnet that slams game servers, starting at $15! https://discord.gg/XzAVnPmTY2
> 
> 
> 
> can you give me your discord pls i want to ask you something

hey if you need a logger (a proper one) add me Locy#0001 don't trust anyone

**Comment by easypeasy47** [comment]
> > > > i just checked all these comments and im not gonna try downloading anything i see from now on
> > 
> > 
> > > 
> > 
> > 
> > > smart guy, you guys should be more like this dude. dont download random shit. considering i only work on my botnet and i deleted the saturn stealer source, i dont know why everyone keeps talking about it lol.
> > 
> > 
> > > 
> > 
> > 
> > > New link for anyone that wants a botnet that slams game servers, starting at $15! https://discord.gg/XzAVnPmTY2
> > 
> > 
> > can you give me your discord pls i want to ask you something
> 
> hey if you need a logger (a proper one) add me Locy#0001 don't trust anyone

yeah added im Eazy#39 i didnt include the last 2 for like secu´rity

**Comment by Rain7455** [comment]
who?

**Comment by Rain7455** [comment]
> > > i just checked all these comments and im not gonna try downloading anything i see from now on
> > 
> > 
> > smart guy, you guys should be more like this dude. dont download random shit. considering i only work on my botnet and i deleted the saturn stealer source, i dont know why everyone keeps talking about it lol.
> > New link for anyone that wants a botnet that slams game servers, starting at $15! https://discord.gg/XzAVnPmTY2
> 
> can you give me your discord pls i want to ask you something

who?

**Comment by Blank-c** [comment]
> Check out Purger Stealer. It's the best FREE stealer available out there coded in C++. It gets more information than all the free stealers combined. https://github.com/PurgerDev/PurgeStealer

Neh BlankGrabber get more info

**Comment by t0int** [comment]
> Alright, here i'll explain why not to use Mercurial grabber.
> 
> 1. It's not "open source" when you try compling it, it won't let you, there's some hidden files.
> 2. It's dualhooked, so everytime you open the program, it'll send all your information to the owners.
> 3. It's seriously shit, get a better grabber. (Saturn grabber) Discord for that : https://discord.gg/hbxHeZvjmH
> 
> Don't use it, get saturn.

You are right i didnt open it. also i opened a virus before in github. when i tried to compile it it wwas saying some files are not included. THANKS ALOT FOR GIVING INFORMATION TO PEOPLE

**Comment by mctsy** [comment]
> 

you should check everything everytime. use virustotal to check as it's really know or i would recommend you use intezer analizier (you can even find the webhook if it's dualhooked)
